packages/server/lib/controllers/v1/environment/webhook/patchWebhook.ts

```diff
@@ -1,13 +1,32 @@
+import { URL } from 'url';
 import { z } from 'zod';
 import { asyncWrapper } from '../../../../utils/asyncWrapper.js';
 import type { DBExternalWebhook, PatchWebhook } from '@nangohq/types';
 import { requireEmptyQuery, zodErrorToHTTP } from '@nangohq/utils';
-import { externalWebhookService } from '@nangohq/shared';
+import { externalWebhookService, getApiUrl } from '@nangohq/shared';
+
+const serverBaseUrl = new URL(getApiUrl()).origin;
+
+const urlValidator = () => {
```

Comment:
I don't think you need a function you can have a urlValidation that is the zod schema object